### ISSUE TITLE
Digital Miner Auto-Pull

### ISSUE THREAD

**Issue by duncanwebb** [aidancbrady/Mekanism#4044]
Is auto pull working on the digital miner, with version Mekanism-1.10.2-9.2.1.295?

Cheers

**Comment by Ricket** [comment]
Can you hop into a creative world and test it?

**Comment by duncanwebb** [comment]
As far as I can tell it is not working.

What I was doing was some landscaping in Desert Hills, it is interesting as sand falls down and get missed by the miner as it is digging from bottom to top in chunks. this does not matter, just reset and restart. However on the last level instead of mining sand and sandstone, I was just mining sand and set the replace in the filter and auto pull on. On top of the miner I placed some transporters connecting to the front and a bin with sandstone. The miner didn't pull the sandstone.

I've tries this both in survival and creative.

![2017-01-05_14 42 11](https://cloud.githubusercontent.com/assets/4198031/21682557/8fe606f4-d355-11e6-878d-ac1450c3386c.png)
![2017-01-05_14 42 59](https://cloud.githubusercontent.com/assets/4198031/21682558/9364a0f6-d355-11e6-82d0-adb8bbd095dc.png)
![2017-01-05_14 43 48](https://cloud.githubusercontent.com/assets/4198031/21682562/965081ae-d355-11e6-9e8f-4a45d54bb548.png)

**Comment by VT-14** [comment]
Try putting the chest/bin directly on top of the Digital Miner.

**Comment by VT-14** [comment]
Picture of the actual filter settings?

**Comment by duncanwebb** [comment]
![2017-01-05_18 20 23](https://cloud.githubusercontent.com/assets/4198031/21690260/bb42b126-d373-11e6-9abb-a79b6c96c990.png)

Does this help?

**Comment by Ricket** [comment]
In your last screenshot, above the ! button on the right is an empty square. I think you're supposed to put the block that you want to replace with, in there. Try putting a sandstone there.

**Comment by VT-14** [comment]
Yup, that's why I asked.  The box above the require replace button is where you set what the block is replaced by.  It should pull from an inventory on top directly.

**Comment by duncanwebb** [comment]
Thanks that fixed it.

![2017-01-05_19 14 54](https://cloud.githubusercontent.com/assets/4198031/21692073/7bad9014-d37b-11e6-9160-7849e62a767f.png)
